modules/nf-core/ensemblvep/main.nf

```diff
@@ -17,12 +17,9 @@ process ENSEMBLVEP {
     path  extra_files
 
     output:
-    tuple val(meta), path("*.ann.vcf")     , optional:true, emit: vcf
-    tuple val(meta), path("*.ann.tab")     , optional:true, emit: tab
-    tuple val(meta), path("*.ann.json")    , optional:true, emit: json
-    tuple val(meta), path("*.ann.vcf.gz")  , optional:true, emit: vcf_gz
-    tuple val(meta), path("*.ann.tab.gz")  , optional:true, emit: tab_gz
-    tuple val(meta), path("*.ann.json.gz") , optional:true, emit: json_gz
+    tuple val(meta), path("*.ann.vcf.gz")  , optional:true, emit: vcf
```

Comment:
Could you also remove the `.ann` suffix? This should be specified with the `prefix` (more flexibility, yay!)

Reply:
good point there!